gui/pages/kernel_options.go

```diff
@@ -0,0 +1,292 @@
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package pages
+
+import (
+	"github.com/clearlinux/clr-installer/gui/common"
+
+	"github.com/clearlinux/clr-installer/kernel"
+	"github.com/clearlinux/clr-installer/log"
+	"github.com/clearlinux/clr-installer/model"
+	"github.com/clearlinux/clr-installer/utils"
+	"github.com/gotk3/gotk3/gtk"
+	"strings"
+)
+
+// KConfigPage is a page to change kernel installation configuration
+type ConfigKernelPage struct {
+	controller Controller
+	model      *model.SystemInstall
+	box        *gtk.Box
+	addEntry   *gtk.Entry
+	remEntry   *gtk.Entry
+	addLabel   *gtk.Label
+	remLabel   *gtk.Label
+	scroll     *gtk.ScrolledWindow
+	list       *gtk.ListBox
+	data       []*kernel.Kernel
+	selected   *kernel.Kernel
+}
+
+// NewConfigKernelPage returns a new NewConfigKernelPage
+func NewConfigKernelPage(controller Controller, model *model.SystemInstall) (Page, error) {
+	kernelArgsHelp :=
+		utils.Locale.Get("Note: The boot manager tool will first include the \"Add Extra Arguments\" items.")
+	kernelArgsHelp = kernelArgsHelp + "\n" +
+		utils.Locale.Get("Then, the \"Remove Arguments\" items are removed.")
+	kernelArgsHelp = kernelArgsHelp + "\n" +
+		utils.Locale.Get("The final argument list contains the kernel bundle's configured arguments and the ones configured by the user.")
+
+	data, err := kernel.LoadKernelList()
+	if err != nil {
+		return nil, err
+	}
+	page := &ConfigKernelPage{
+		controller: controller,
+		model:      model,
+		data:       data,
+	}
+
+	// Box
+	page.box, err = setBox(gtk.ORIENTATION_VERTICAL, 0, "box-page")
+	if err != nil {
+		return nil, err
+	}
+
+	// ScrolledWindow
+	page.scroll, err = setScrolledWindow(gtk.POLICY_NEVER, gtk.POLICY_AUTOMATIC, "scroller")
+	if err != nil {
+		return nil, err
+	}
+	page.scroll.SetMarginStart(common.StartEndMargin)
+	page.scroll.SetMarginEnd(common.StartEndMargin)
+	page.box.PackStart(page.scroll, true, true, 0)
+
+	// ListBox
+	page.list, err = setListBox(gtk.SELECTION_SINGLE, true, "list-scroller")
+	if err != nil {
+		return nil, err
+	}
+	if _, err := page.list.Connect("row-activated", page.onRowActivated); err != nil {
+		return nil, err
+	}
+	page.scroll.Add(page.list)
+
+	// Create list data
+	for _, v := range page.data {
+		name := v.Name
+		desc := v.Desc
```

Comment:
Please add a call to the util localization to the description of the kernel and add entries to the po files.
The name may not make sense to translate, but if we leave that in English having the description translated would be helpful.

Reply:
Done.